test/behavior/floatop.zig

```diff
@@ -697,3 +697,38 @@ test "f128 at compile time is lossy" {
 
     try expect(@as(f128, 10384593717069655257060992658440192.0) + 1 == 10384593717069655257060992658440192.0);
 }
+
+test "f16 literals" {
+    if (builtin.zig_backend != .stage2_c) return error.SkipZigTest; // TODO
+
+    var buf: f16 = math.f16_min;
+    try expect(math.f16_min == buf);
+
```

Comment:
missing test cases +-inf, -0 with at least 1 example with negative sign.
Not sure, if you also want to check `f16_true_min` or what the expected behavior for denormals is.